apps/splash-2025/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/png" href="/online-logo.png"/>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Kvitteringsskjema</title>
+  <title>Fadderukene 2025</title>
   <script defer data-domain="kvittering.online.ntnu.no" src="https://plausible.io/js/script.js"></script>
```

Comment:
Unrelated to this PR, but this line should probably removed @henrikskog

Reply:
Haha yepp!